### ISSUE TITLE
Add another disk to the VM

### ISSUE THREAD

**Issue by marlluslustosa** [terra-farm/terraform-provider-xenorchestra#33]
How can I add a new disk to the VM, apart from the current disk?
My case: Create a VM and make it have two disks at startup.

`main.tf:`

```

...

resource "xenorchestra_vm" "bar" {
    memory_max = 1073741824
    cpus  = 1
    cloud_config = "${xenorchestra_cloud_config.bar.template}"
    name_label = "VM-build-Terraform"
    name_description = "VM criada a partir do terraform - usando provider Xen Orchestra"
    template = "${data.xenorchestra_template.template.id}" #posso tambem colocar o uuid do template
    network {
      network_id = "${data.xenorchestra_pif.pif.network}"
    }

    disk {
      sr_id = "0c931761-7885-52c3-13ba-3dddd4dd84b2"
      name_label = "Debian10-terraform"
      size = 4294967296
    }

   #other disk here
    disk {
      sr_id = "0c931761-7885-52c3-13ba-3dddd4dd84b2"
      name_label = "Debian10-terraform-1"
      size = 11212254720
    }
}

```

it does not fail after `terraform apply`, however, the other disk is neither created nor attached to the VM.

this is normal? Any ideas / suggestions here?

**Comment by ddelnano** [comment]
@marlluslustosa I have never tested that functionality out before with the `xenorchestra_vm` resource. The documentation should do a better job explaining what is / isn't tested or supported (which I can update through this as well).

I will need to take a look at how we can enhance the resource to take into account all disks, make sure they are attached to the VM and that terraform can correctly manage the state of the resource once it's created (showing a "diff" when a disk has been detached, etc).

**Comment by ddelnano** [comment]
This will be fixed in v0.9.0 of the provider (released later today).

**Comment by marlluslustosa** [comment]
thanks for the effort!
